src/components/dev-hub/mongodb-live-banner.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import useMedia from '../../hooks/use-media';
+import Image from '../Image';
+import Link from './link';
+import { screenSize } from './theme';
+
+const LiveImage = styled(Image)`
+    border-radius: 0;
+    margin-bottom: 0;
+    width: 100%;
+`;
+
+const MongodbLiveBanner = () => {
+    const isMobile = useMedia(screenSize.upToMedium);
+    return (
+        <Link href="https://www.mongodb.com/resources/channel/mongodblive">
+            <LiveImage
+                alt="Click to watch keynotes and sessions from MongoDB.live, our virtual developer conference."
+                src={
+                    isMobile
+                        ? '/public/images/dot-live-sm.png'
```

Comment:
do you have any cacheing issues with using the same name?

Reply:
Oh good point! I didn't see anything locally but it may be worth changing to avoid any potential issues.